packages/connect/src/ws_orderbook_channel.ts

```diff
@@ -46,23 +47,25 @@ export class WebSocketOrderbookChannel implements OrderbookChannel {
         assert.isFunction('handler.onUpdate', _.get(handler, 'onUpdate'));
         assert.isFunction('handler.onError', _.get(handler, 'onError'));
         assert.isFunction('handler.onClose', _.get(handler, 'onClose'));
+        this.subscriptionCounter += 1;
         const subscribeMessage = {
             type: 'subscribe',
             channel: 'orderbook',
+            requestId: this.subscriptionCounter,
```

Comment:
Nit: I prefer `this.subscriptionCounte++`, but I think @fabioberger likes your style more ;)